### ISSUE TITLE
Affichage des hauteurs d'activation

### ISSUE THREAD

**Issue by JeromeLeCoz** [BaRatin-tools/BaRatinAGE#11]
salut

un détail un peu gênant sur l'affichage des "k" sur les plots de "RC", c'est que quand deux enveloppes se recouvrent, l'effet de transparence donne le même vert (violet) que celui de la valeur MaxPost (MaxPrior). Ca complique un peu l'interprétation, exemple : 
![image](https://user-images.githubusercontent.com/86300732/204227255-d6cb2e2c-eed1-4977-b9be-23f93a210d6c.png)

Serait-il possible de rendre les enveloppes encore plus transparentes pour éviter cet effet, ou encore d'afficher les MaxPost/MaxPrior avec une autre couleur? (genre noir comme la CT MaxPost/MaxPrior? en pointillés peut-être pour ne pas détourner l'attention de la RC?)

**Comment by IvanHeriver** [comment]
I took your suggestion and increased the contrast between the envelop color and the maxpost color so there cannot be any confisusion between envelops overlap and maxpost... Let me know if the changes are ok so I can close this issue.

**Comment by IvanHeriver** [comment]
I made the transition lines dashed and smaller (see commit b069eab9da86d3091074d0102d4948b73c7cd854)

**Comment by IvanHeriver** [comment]
@JeromeLeCoz , can you try with [this jar file](https://ivanheriver-drive.mycozy.cloud/public?sharecode=zwWL0SorMAhg) .

You simply need to replace the jar file with the same name in the app folder of the packaged version.

**Comment by JeromeLeCoz** [comment]
I love the new display, thx!

However, I confirm running prior and posterior RCs fail with this version 2-2 beta1... There is a pb.

**Comment by JeromeLeCoz** [comment]
The legends of the Prior and Posterior RC must be updated:

Prior RC legend: Activation stage uncertainty in purple. I suggest adding the dashed purple line for "Prior activation stage"

Posterior RC legend: I suggest adding the dashed green line for "Prior activation stage"

**Comment by JeromeLeCoz** [comment]
Tant qu'à faire, autant enlever la barre d'incertitude horizontale (en H) des jaugeages, sur la légende.

**Comment by IvanHeriver** [comment]
By the way, I find it very weird to have the button "show legend" open an image file... I would expect the legend to be added next to the plot or a popup within BaRatinAGE containing the legend (e.g. similar to what happens when you click "external figure")...

Also, this means that one image per language must be created (wasted spaces, time consuming, difficult to track) and the legend should not be related to the help/documentation (as it is the case today)...

Basically, I think the legend should be created dynamically within BaRatinAGE and use translated text.

What are your thoughts on this @benRenard  and @JeromeLeCoz ? Should I open a new issue to address this in v3?

**Comment by benRenard** [comment]
Fully agree! worth a ticket for v3.

**Comment by JeromeLeCoz** [comment]
About the proposed legend:

- keep the same terms used in BaRatinAGE: gaugings (not stage-discharge measurements)
- I don't like separating gauging dot and uncertainty bar (waste of space, complexity): I prefer the current display
- I like the combination of activation stage and its 95% uncertainty in the same symbol (for same reason), but then the caption should be "activation stage and its uncertainty"
- the "Prior uncertainty" is a parametric uncertainty but I guess "Prior uncertainty" is correct and easier to understand (and avoid confusion with posterior RC uncertainty). Maybe "Prior RC uncertainty"?

**Comment by IvanHeriver** [comment]
> - keep the same terms used in BaRatinAGE: gaugings (not stage-discharge measurements)
> - I don't like separating gauging dot and uncertainty bar (waste of space, complexity): I prefer the current display

Ok, not a problem. Note That I may have misunderstand and modify this legend although it wasn't needed and maybe even not desirable at all: this legend is for the **_gauging panel_** (which was not modified). I just copy/pasted what was in the previous legend but removed the stage uncertainties (i.e. the horizontal uncertainty bar)...

Two options:
 1. revered back to the previous legend
 2. keep this new version but use "Gauging" instead and merge with uncertainty: so the legend would read `Gauging and its discharge uncertainty (95% interval)`

---

> - I like the combination of activation stage and its 95% uncertainty in the same symbol (for same reason), but then the caption should be "activation stage and its uncertainty"

OK. I suggest the label: `Activation stage and its uncertainty (95% inteval)` to be consistent (with the "95% interval" in between parentheses written this way everywhere else)

---
> - the "Prior uncertainty" is a parametric uncertainty but I guess "Prior uncertainty" is correct and easier to understand (and avoid confusion with posterior RC uncertainty). Maybe "Prior RC uncertainty"?

OK. I like `Prior rating curve uncertainty (95% interval)`. Longer but no confusion possible.

**Comment by IvanHeriver** [comment]
Please check out [here](https://github.com/BaRatin-tools/BaRatinAGE/commit/0a2dabee0bdac980ca4ab96043eacbd09d2011b1) to see all the changes made to the legends.

Note that I aslo added the svg source files.

**Comment by IvanHeriver** [comment]
ok greate! Closing this issue then.
